ParseTree/ParseTree/Tree.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace ParseTree;
+
+using System.Text;
+
+/// <summary>
+/// Class for building a parse tree.
+/// </summary>
+public class Tree
+{
+    /// <summary>
+    /// Method for building a parse tree.
+    /// </summary>
+    /// <param name="filePath">Path to the file.</param>
+    /// <returns>Root of parse tree.</returns>
+    public static Node BuildTree(string filePath)
+    {
+        StreamReader reader = new(filePath);
```

Comment:
```suggestion
        uasing var reader = new(filePath);
```
Чтобы потом ресурсы освободить